relay-general/src/processor/selector.rs

```diff
@@ -138,22 +138,24 @@ pub enum SelectorSpec {
     Or(Vec<SelectorSpec>),
     Not(Box<SelectorSpec>),
     Path(Vec<SelectorPathItem>),
+    Other(String),
 }
 
 impl fmt::Display for SelectorSpec {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        match *self {
+        match self {
             SelectorSpec::And(ref xs) => {
                 for (idx, x) in xs.iter().enumerate() {
                     if idx > 0 {
                         write!(f, " && ")?;
                     }
 
-                    let needs_parens = match *x {
+                    let needs_parens = match x {
```

Comment:
I got borrow-checking errors if I didn't do this, something about the string in Other() being moved out, I don't know exactly how this works

Reply:
What is the error, could you maybe add it here?